crates/rattler_repodata_gateway/Cargo.toml

```diff
@@ -13,6 +13,7 @@ readme.workspace = true
 [dependencies]
 async-compression = { workspace = true, features = ["gzip", "tokio", "bzip2", "zstd"] }
 blake2 = { workspace = true }
+bytes = "1.6.0"
```

Comment:
Since `sparse` is a feature, we should only enable this when the `sparse` feature is used.

Reply:
Oh and also please add it as a workspace dependency

Reply:
Done!